resources/views/common/skill.html.twig

```diff
@@ -572,8 +572,8 @@
                                     {# Word Counter #}
                                     {# <div
                                         data-word-counter-id={{ "#{id_prefix}skillDescription"|json_encode() }}
-                                        data-max-words={{ "100"|json_encode()  }}
-                                        data-min-words={{ "75"|json_encode()  }}
+                                        data-max-words={{ "80"|json_encode()  }}
```

Comment:
We'll want to set the max and min words in the same place we create the messages, (speaking of which, we'll need translations for those) which probably means in a very context-specific SkillDeclarationWordCounter component. This component will need to pass in all the constants to WordCounterWrapper and also mount the component with ReactDOM.